sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -244,23 +232,16 @@ boolean deleteOnline(String table, String partition, String documentId) {
                         values.getAsString(PARTITION_COLUMN_NAME),
                         values.getAsString(DOCUMENT_ID_COLUMN_NAME),
                         values.getAsString(DOCUMENT_COLUMN_NAME),
-                        values.getAsLong(EXPIRATION_TIME_COLUMN_NAME)));
+                        values.getAsLong(EXPIRATION_TIME_COLUMN_NAME),
+                        values.getAsLong(DOWNLOAD_TIME_COLUMN_NAME),
```

Comment:
I can't find where it is used for the initial put.

Reply:
Also it's read but never used.

Reply:
The sqlite database schema contains `download_time` and `operations_time`:
```
INTEGER oid (primary key auto increment)
   TEXT partition
   TEXT document_id
   TEXT document
   TEXT etag
INTEGER expiration_time
INTEGER download_time
INTEGER operation_time
   TEXT pending_operation (NULL, CREATE, REPLACE, DELETE)
```

The current `PendingOperation` model contains all the columns except `download_time` and `operations_time` columns. As part of this PR, we need to call [`mLocalDocumentStorage.updatePendingOperation(pendingOperation)`](https://github.com/Microsoft/AppCenter-SDK-Android/pull/1091/files#diff-9a1784d794428d5c4bddb2690a20dcb5R906) which internally calls `mDatabaseManager.replace` API. That is why we need the 1:1 mapping of database columns to `PendingOperation` properties to be able to replace an row in database. Without such mapping, I believe it would set `download_time` and `operations_time` columns to NULL.